exo-add/features/steps/steps-when.ls

```diff
@@ -42,6 +42,17 @@ module.exports = ->
       ..on 'ended', done
 
 
+  @When /^trying to run "([^"]*)" in this application's directory$/, timeout: 600_000, (command) ->
```

Comment:
Let's standardize the nomenclature: 
- _starting_ a process means kicking it off and continuing the test while the process is running
- _running_ a process means starting the process and waiting until it has finished. The process should finish without errors, meaning if it finishes with an error, this step fails.
- _trying to run_ a task means running a process, waiting until it is finished, and verifying that it has failed. If the process finished without errors, this step fails.

So, with that in mind, this step must wait for the process to finish, and then verify that `@process.exit-code` is > 0.